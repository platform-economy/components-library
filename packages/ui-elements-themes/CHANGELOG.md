# Change Log

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [0.1.2](https://github.com/relayr/ui-elements/compare/@relayr/ui-elements-themes@0.1.1...@relayr/ui-elements-themes@0.1.2) (2019-08-22)

**Note:** Version bump only for package @relayr/ui-elements-themes






## [0.1.1](https://github.com/relayr/ui-elements/compare/@relayr/ui-elements-themes@0.1.0...@relayr/ui-elements-themes@0.1.1) (2019-08-21)

**Note:** Version bump only for package @relayr/ui-elements-themes






# 0.1.0 (2019-08-02)


### Features

* **ui-elements-basic:** new InputLabel component ([abb6076](https://github.com/relayr/ui-elements/commit/abb6076))
* **ui-elements-layout:** add Card component ([e8db0cf](https://github.com/relayr/ui-elements/commit/e8db0cf))
* **ui-elements-tables:** Table components ([#102](https://github.com/relayr/ui-elements/issues/102)) ([b1b8d6f](https://github.com/relayr/ui-elements/commit/b1b8d6f))
* **ui-elements-themes:** custom font linking system ([8e9f0f0](https://github.com/relayr/ui-elements/commit/8e9f0f0))
