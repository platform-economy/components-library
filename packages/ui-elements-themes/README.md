# ui-elements-themes

## Add to project

via npm:

```bash
npm install --save @relayr/ui-elements-themes
```

or Yarn:
```bash
yarn add @relayr/ui-elements-themes
```
