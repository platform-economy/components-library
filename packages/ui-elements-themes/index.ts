export * from './themes/light';
export * from './themes/dark';
export * from './themes/test';
export * from './themes/_common/paletteDark';
export * from './themes/_common/paletteLight';
export * from './themes/_common/methods';
export * from './themes/_common/fonts';
export * from './themes/_common/withTestTheme';
