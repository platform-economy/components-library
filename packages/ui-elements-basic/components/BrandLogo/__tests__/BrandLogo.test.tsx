import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
import 'jest-styled-components';

import { BrandLogo } from '../index';

import { withTestTheme } from '@relayr/ui-elements-themes';

describe('ui-elements-basic', () => {
  describe('Components', () => {
    describe('BrandLogo', () => {
      it('should render with an img property', () => {
        const tree = TestRenderer
          .create(withTestTheme((
          <BrandLogo img="http://tv.proximetry.com/assets/images/relayr_logo_white.png" />
          )))
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
      it('should render without an img property', () => {
        const tree = TestRenderer
          .create(withTestTheme((
          <BrandLogo />
          )))
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
      it('should render smaller if isMobile is set', () => {
        const tree = TestRenderer
          .create(withTestTheme((
          <BrandLogo isMobile={true} />
          )))
          .toJSON();
        expect(tree).toMatchSnapshot();
      });
    });
  });
});
