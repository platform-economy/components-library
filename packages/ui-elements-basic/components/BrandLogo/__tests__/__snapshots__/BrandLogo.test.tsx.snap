// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-elements-basic Components BrandLogo should render smaller if isMobile is set 1`] = `
.c0 {
  display: block;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  margin: 0;
  border: 15px solid transparent;
  box-sizing: border-box;
  background: center / contain no-repeat url("http://testUrl");
  width: 100px;
  height: 64px;
}

<div
  className="c0"
/>
`;

exports[`ui-elements-basic Components BrandLogo should render with an img property 1`] = `
.c0 {
  display: block;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  margin: 0;
  border: 15px solid transparent;
  box-sizing: border-box;
  background: center / contain no-repeat url("http://tv.proximetry.com/assets/images/relayr_logo_white.png");
  width: 200px;
  height: 64px;
}

<div
  className="c0"
/>
`;

exports[`ui-elements-basic Components BrandLogo should render without an img property 1`] = `
.c0 {
  display: block;
  -webkit-flex: none;
  -ms-flex: none;
  flex: none;
  margin: 0;
  border: 15px solid transparent;
  box-sizing: border-box;
  background: center / contain no-repeat url("http://testUrl");
  width: 200px;
  height: 64px;
}

<div
  className="c0"
/>
`;
